javagen/src/main/java/com/azure/autorest/model/javamodel/JavaClass.java

```diff
@@ -199,6 +204,64 @@ public final void javadocComment(int wordWrapWidth, Consumer<JavaJavadocComment>
         contents.javadocComment(wordWrapWidth, commentAction);
     }
 
+    @Override
+    public final void javadocComment(int wordWrapWidth, Consumer<JavaJavadocComment> commentDescriptionAction, Consumer<JavaJavadocComment> commentTagsAction, boolean withGeneratedMarker) {
```

Comment:
Can we make `Consumer<JavaJavadocComment> commentDescriptionAction, Consumer<JavaJavadocComment> commentTagsAction` to `Consumer<JavadocDescription> commentDescriptionAction, Consumer<JavadocTags> commentTagsAction`?

Then no one need to remember which parameter is for which. The type tells everything.

And `JavadocDescription` would only take the description, while `JavadocTags` takes parameter, return, throw etc. And one cannot add description on tags block, or tags to description block.

Also prefer to have overload of `javadocComment(int wordWrapWidth, Consumer<JavaJavadocComment> commentDescriptionAction, boolean withGeneratedMarker)` for cases that no tag is needed.

Reply:
BTW, not sure if we really need the `boolean withWordWrap` or `int wordWrapWidth` param anymore.

Maybe we can clean them up in the new APIs?

PS: I have no problem drop `boolean withGeneratedMarker` as well. Just make it always true is OK for me.

Reply:
I'm fine to drop `withWordWrap`, as long as everyone agrees, as there is some code diff if we remove this.
`withGeneratedMarker `: I will keep it first because currently it is always `false`, and we can merge the pr first and generated code will be the same.

Reply:
We should also drop `wordWrapWidth` as we eventually will use Spotless codeformatter to make it consistent with the monorepo.